src/style-anchorlink.scss

```diff
@@ -3,22 +3,24 @@
 
 // ---------------------------------------------------------------------
 
+/* stylelint-disable declaration-empty-line-before, declaration-no-important, order/properties-order, max-line-length, scss/double-slash-comment-empty-line-before */
+
 @import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
 
 // Import Auro tokens
 @import './../node_modules/@aurodesignsystem/webcorestylesheets/src/breakpoints';
 @import './../node_modules/@aurodesignsystem/webcorestylesheets/src/core';
 
-/* stylelint-disable declaration-empty-line-before, declaration-no-important, order/properties-order, max-line-length */
-
 :host {
   display: block;
-  border: 1px solid transparent;
-  border-left: 1px solid var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  border-width: 1px;
+  border-style: solid;
+
+  border-left-width: 1px;
+  border-left-style: solid;
```

Comment:
why re-declare these when they are technically already set with the same values on line 16-17?